test/langtools/jdk/javadoc/doclet/testLinkTaglet/TestLinkTagletPrimitive.java

```diff
@@ -88,6 +88,64 @@ private C() { }
                     </div>
                     """);
     }
+
+    @Test
+    public void testSimpleDocLint(Path base) throws IOException {
+        Path src = base.resolve("src");
+
+        tb.writeJavaFiles(src, """
+                /**
+                 * Comment.
+                 * Double: {@link double}
+                 * Void: {@link void}
+                 * @see int
```

Comment:
Consider adding a case of a primitive array, which while relates to primitives is also an object.

Reply:
> Consider adding a case of a primitive array, which while relates to primitives is also an object.

Oh, sorry, I see you did it in a separate test. I don't know why I didn't notice it. Sigh.